src/common/components/modal/modal.module.scss

```diff
@@ -32,35 +36,42 @@ $modalPadding: 2rem;
   justify-content: center;
 }
 
+.closeButton {
+  float: right;
+  @include respond-below(md) {
+    z-index: 1;
+    position: relative;
+    text-align: right;
+    width: 100%;
+  }
+  button {
+    padding: 0;
+    border: none;
+    height: $largeMargin;
+    width: $largeMargin;
+    font-weight: $font-weight-bold;
```

Comment:
The x button is not a character, its an image, so this is useless (try to toggle it on and off)

You should control the image size instead if you expect the same effect